src/utils/parser/getSymbols.js

```diff
@@ -84,9 +90,217 @@ export default function getSymbols(source: SourceText): SymbolDeclarations {
           location: path.node.loc
         });
       }
+
+      if (t.isObjectProperty(path)) {
+        const { start, end, identifierName } = path.node.key.loc;
+        symbols.objectProperties.push({
+          name: identifierName,
+          location: { start, end },
+          expression: getExpression(path)
+        });
+      }
+
+      if (t.isMemberExpression(path)) {
+        const { start, end } = path.node.property.loc;
+        symbols.memberExpressions.push({
+          name: path.node.property.name,
+          location: { start, end },
+          expressionLocation: path.node.loc,
+          expression: getExpression(path)
+        });
+      }
+
+      if (t.isIdentifier(path)) {
+        const { start, end } = path.node.loc;
+
+        symbols.identifiers.push({
+          name: path.node.name,
+          expression: path.node.name,
+          location: { start, end }
+        });
+      }
+
+      if (t.isVariableDeclarator(path)) {
+        const node = path.node.id;
+        const { start, end } = path.node.loc;
+
+        symbols.identifiers.push({
+          name: node.name,
+          expression: node.name,
+          location: { start, end }
+        });
+      }
     }
   });
 
   symbolDeclarations.set(source.id, symbols);
   return symbols;
 }
+
+function addProperty(name, expression, path, prevPath) {
+  const computed = path && path.node.computed;
+  const prevComputed = prevPath && prevPath.node.computed;
+  const prevArray = t.isArrayExpression(prevPath);
+  const array = t.isArrayExpression(path);
+
+  if (expression === "") {
+    if (computed) {
+      return `[${name}]`;
+    }
+    return name;
+  }
+
+  if (computed || array) {
```

Comment:
i dont think we need this block, since it is doing the same thing as lines 160 - 164, and if it is computed -- we have an early return at line 150